Composer/packages/ui-plugins/virtual-assistant-creation/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@bfc/ui-plugin-va-creation",
+  "version": "0.0.0",
```

Comment:
Consider 0.0.1

Reply:
I was following the standard set in the other ui plugin workspaces. Is there a functional reason they are all 0.0.0? When these get referenced in client package.json they have the "*" version indicator